packages/@react-spectrum/theme-light/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@react-spectrum/theme-light",
+  "version": "3.1.0",
```

Comment:
Wasn't 100% about the version, both default and dark themes are in `3.1.0` so made it match, down to change it to whatever though!

Reply:
This is fine. Our release scripts will take care of correcting it when it goes out! 😄